web-extensions/resources/runtime/background.js

```diff
@@ -0,0 +1,64 @@
+async function testGetURLErrorCases() {
```

Comment:
I think usually the actual WPT tests are added separately from the infrastructure changes and tests for the infrastructure should be added to https://github.com/web-platform-tests/wpt/tree/master/infrastructure